Sources/MapboxNavigation/ImageRepository.swift

```diff
@@ -44,19 +43,14 @@ class ImageRepository {
             return
         }
 
-        let _ = imageDownloader.download(with: imageURL, completion: { [weak self] (cachedResponse, error) in
+        let _ = imageDownloader.download(with: imageURL, completion: { [weak self] result in
```

Comment:
```suggestion
        imageDownloader.download(with: imageURL, completion: { [weak self] result in
```
`download` returns nothing, so we can remove empty assignment